chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimitVsBaseline.java

```diff
@@ -16,28 +17,53 @@ private JLimitVsBaseline(Builder builder) {
     /**
      * Builder for {@link JLimit} to compare with baseline.
      *
-     * @param metricName metric name.
+     * @param metricId metric name.
      * @return builder for {@link JLimit}.
      */
-    public static Builder builder(String metricName) {
-        return new Builder(metricName);
+    public static Builder builder(String metricId) {
+        return new Builder(metricId);
     }
-
+    
+    
+    /**
+     * Builder for {@link JLimit} to compare with current value.
```

Comment:
to compare metrics, collected during {@link JHttpUserScenario} execution, with the baseline